src/Blocks/BlockHandler.h

```diff
@@ -25,9 +25,11 @@ class BlockTypeRegistry;
 class cBlockHandler
 {
 public:
-	cBlockHandler(BLOCKTYPE a_BlockType);
 
-	virtual ~cBlockHandler() {}
```

Comment:
This should still have a virtual destructor, even if you don't think you'll need it.

Reply:
aww but I looked forward to deleting this. I'm not doing any dynamic allocation, so why do I need one?

Reply:
AFAIK there's no reason not to include it, and some potentially nasty bugs avoided by including it.

Reply:
Hm, it doesn't let me add a destructor to constexpr

Reply:
Hrm, I guess it's not possible for `constexpr` then. Oh well.